LongRunningIntegrationTestDs3/LongRunningIntegrationTestDs3.cs

```diff
@@ -90,20 +105,42 @@ public void TestChecksumStreamingWithMultiChunks()
                 var directoryObjects = new List<Ds3Object> { new Ds3Object("bigFile", streamLength) };
 
                 const long blobSize = 1L * 1024L * 1024L * 1024L;
-                var job = _helpers.StartWriteJob(bucketName, directoryObjects, blobSize, new WriteStreamHelperStrategy());
 
-                var md5 = MD5.Create();
-                var fileStream = new ChecksumStream(streamLength, this._copyBufferSize.Value);
-                var md5Stream = new CryptoStream(fileStream, md5, CryptoStreamMode.Read);
+                // Test the PUT
```

Comment:
there are 2 tests, each one of them does PUT and than does GET. both of them using the stream strategy